### PR TITLE
ECMPS beta examples

### PR DESCRIPTION
**Pull request by kstetsyuk**

### CODE REVIEW

File: .github/workflows/jekyll-gh-pages.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup Pages
+        uses: actions/configure-pages@v3
+      - name: Setup & Configure Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16
```
Comment (j-tafoya) [inline .github/workflows/jekyll-gh-pages.yml:36]: May want to increase this version, since node 16 has reached EOL.

**Review by j-tafoya — CHANGES_REQUESTED** [review]
This looks good! See my comments in the files for more feedback. I think the only thing to change may be the node version. I really like the format of the tables and the light dependencies!

**Comment by kstetsyuk** [comment]
I agree on trying to not duplicate code. I'm sure there's something out there we could use to remediate. Other than that, changes are made! @j-tafoya

**Review by j-tafoya — APPROVED** [review]
👍

**Pull request merged**